boards/DCM/Inc/App/App_Imu.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <App_SharedExitCode.h>
+
+struct Imu;
+
+/**
+ * Allocate and initialize an IMU.
```

Comment:
make sure IMU is consistent thoughout the file

Reply:
My bad forgot about this, should be fixed now